src/activate/activate.c

```diff
@@ -151,7 +151,7 @@ static int exec_process(const char *name, char **argv, char **env, int start_fd,
 
                         envp[n_env++] = k;
                 } else {
-                        _cleanup_free_ char *p;
+                        _cleanup_free_ char *p = NULL;
```

Comment:
About the commit message:
> It doesn't cost anything to initialize them, so do it.

Not exaaaactly. There is a cost, at least with non-optimized builds. We think that the cost is small enough and worth the benefit, but it is non-zero.